tests/update/updates_packagekit_gpk.pm

```diff
@@ -71,8 +76,14 @@ sub run {
             # Wait until installation is done
             assert_screen \@updates_installed_tags, 3600;
             if (match_has_tag("updates_none")) {
-                send_key "ret";
-                last;
+                send_key 'ret';
+                if (check_screen "updates_installed-restart") {
+                    select_console 'root-console';
+                    type_string "reboot\n";
```

Comment:
bad idea because there are many more things to consider when "just rebooting", e.g. depending on the architecture. Better use `utils::power_action`

Reply:
will adapt this in the whole module then